api/spec/v1.yaml

```diff
@@ -1073,18 +1123,92 @@ components:
+        
+        Consensus supports only staking addresses (`context="oasis-core/address: staking"` below).
+        Runtimes support all types. This means that every consensus address is also
+        valid in every runtime. For example, in EVM runtimes, you can use staking
+        addresses, but only with oasis tools (e.g. a wallet); EVM contracts such as
+        ERC20 tokens or tools such as Metamask cannot interact with staking addresses.
+      type: object
+      properties:
+        context:
+          type: string
+          enum:
+            - "oasis-core/address: staking"
+            - "oasis-runtime-sdk/address: secp256k1eth"
+            - "oasis-runtime-sdk/address: sr25519"
+            - "oasis-runtime-sdk/address: multisig"
+            - "oasis-runtime-sdk/address: module"
+          description: |
+            The method by which the oasis address was derived from `address_data`.
+          example: "oasis-runtime-sdk/address: secp256k1eth"
+        context_version:
+          type: integer
+          nullable: true
+          default: 0
+          description: Version of the `context`.
+        address_data:
+          type: string
+          format: hex
+          description: |
+            The hex-encoded data from which the oasis address was derived. 
+            When `context = "oasis-runtime-sdk/address: secp256k1eth"`, this
+            is the Ethereum address (without the leading `0x`).
+          example: '9907A0cF64Ec9Fbf6Ed8FD4971090DE88222a9aC'
+
+    RuntimeBalance:
+      description: Balance of an account in a runtime.
+      type: object
+      properties:
+        amount:
+          type: string
+          description: Number of base units held; as a string.
+          example: "1000000000000000000000"
+        runtime:
+          type: string
+          enum: ["emerald"]
+          description: Name of the runtime.
+        # TODO:
+        # - How do we handle NFTs? Add another field?
+        # - We do not handle non-native tokens from the `accounts` SDK module. The plan is to use
+        #   their denomination name for both `token_id` and `token_symbol`, or update the SDK to have tickers.
+        token_id:
+          type: string
+          format: binary
+          description: Unique dentifier for the token. For EVM tokens, this is their eth address.
+        token_symbol:
+          type: string
+          description: The token ticker symbol. Not guaranteed to be unique across distinct tokens.
+
     Account:
       type: object
       properties: 
         address:
           type: string
           description: The staking address for this account.
           example: *staking_address_1
+        address_preimage:
+          $ref: '#/components/schemas/AddressPreimage'
         nonce:
           type: integer
           format: int64
           description: A nonce used to prevent replay.
           example: 0
+        # TODO: limit this to 1000 entries. If folks have more,
```

Comment:
👍 I think we should also expose our default limit which I believe is 100.

Reply:
Added to the spec:
```
      default: 100
      minimum: 1
      maximum: 1000
```
Default is [indeed 100](https://github.com/oasisprotocol/oasis-indexer/blob/d56599667bfd4d0d5318f7102a5c493cd5770686/api/common/pagination.go#L16-L16)